src/frontend/EdFi.DataManagementService.Frontend.AspNetCore/appsettings.json

```diff
@@ -3,11 +3,11 @@
         "BeginAllowedSchoolYear": 2022,
         "EndAllowedSchoolYear": 2035,
         "AuthenticationService": "http://dev-test-auth-service",
-        "DatabaseEngine": "mssql",
+        "DatabaseEngine": "postgresql",
         "DeployDatabaseOnStartup": false
     },
     "ConnectionStrings": {
-        "DatabaseConnection": "Data Source=.\\;Initial Catalog=master;Integrated Security=True;"
+        "DatabaseConnection": "host=dmsdb;port=5432;username=postgres;database=edfi_datamanagementservice;"
```

Comment:
This is probably fine, Generally I wouldn't put Docker-specific connection information into the main `appsettings.json`. We can change later if needed - don't worry about it right now.